packages/devextreme/js/ui/tabs.js

```diff
@@ -681,6 +681,17 @@ const Tabs = CollectionWidget.inherit({
         this._toggleFocusedDisabledPrevClass(currentIndex, shouldPrevClassBeSetted);
     },
 
+    _updateFocusedElement() {
+        const { focusStateEnabled, selectedIndex } = this.option();
```

Comment:
Perhaps, we may think about clearing 'focusedElement' option at the right moment. It allows us to use logic from collection that determine correct element that should be focused on focusIn.